### ISSUE TITLE
[Accessibility Feature] Buttons without accessible name exist

### ISSUE THREAD

**Issue by yncat** [cline/cline#4245]
### What happened?

I can't tell some buttons' behaviors because there is no label / accessible name to them. I obviously do not know (visually) which is unlabeled. I will need to look into the source.

### Steps to reproduce

1. Read through Cline's main interface with screen readers.
2. Find inaccessible buttons which are announced as "button" or "icon button".

### Relevant API REQUEST output

```shell

```

### Provider/Model

LiteLLM

### Operating System

Windows

### System Info

Screen readers: NVDA and Narrator tested

### Cline Version

v3.17.13

### Additional context

It should be fixable just by adding aria-label to each of them. I'm happy to contribute.

**Comment by kevinneung** [comment]
Thanks for reporting this issue, and making a PR @yncat ! Your contribution is definitely appreciated, please feel free to continue improving accessibility within Cline. The team will continue reviewing your PR, and we'll let you know if there's any changes needed.

**Comment by kevinneung** [comment]
Closing as we merged your PR
